addon-api/content-script/Addon.js

```diff
@@ -9,8 +9,28 @@ import Tab from "./Tab.js";
 export default class UserscriptAddon extends Addon {
   constructor(info) {
     super(info);
+    this._addonId = info.id;
     this.__path = document.getElementById("scratch-addons").getAttribute("data-path");
     this.tab = new Tab(info);
+    this.self.disabled = false;
+    this.self.late = info.late;
```

Comment:
References to `late` should be removed from this PR, userscript autorefresh should be another PR.